src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -539,20 +529,20 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
 
   override def setStatus(workflowId: WorkflowId, scopeKeys: Traversable[ExecutionDatabaseKey],
                          callStatus: CallStatus): Future[Unit] = {
+    val endTime = if (callStatus.isTerminal) Option(new Timestamp(Calendar.getInstance().getTime.getTime)) else None
```

Comment:
Is there a particular reason why the start date is `new Date().toTimestamp` and the end date is `new Timestamp(Calendar.getInstance().getTime.getTime)` ?